vcpkg.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "creo2urdf",
+  "homepage": "https://github.com/icub-tech-iit/creo2urdf",
+  "documentation": "https://icub-tech-iit.github.io/creo2urdf/doc/doxygen/doc/html/index.html",
+  "dependencies": [
+    "eigen3",
+    "yaml-cpp",
+    "libxml2",
+    "rapidcsv",
+    "idyntree"]
+}
```

Comment:
Could it make sense to add a baseline? See https://github.com/Microsoft/vcpkg-docs/blob/main/vcpkg/consume/lock-package-versions.md#2---add-version-constraints-using-a-baseline .

Reply:
Probably Yes, I removed it to simplify things but it's good for enforcing the idyntree version